web/packages/teleterm/src/ui/ConnectMyComputer/connectMyComputerContext.tsx

```diff
@@ -150,7 +240,7 @@ function isProcessInErrorOrExitState(
 
     return new Error(
       [
-        `Agent process exited with ${codeOrSignal}.`,
+        `Agent process failed to start - the process exited with ${codeOrSignal}.`,
```

Comment:
```suggestion
        `Agent process failed to start – the process exited with ${codeOrSignal}.`,
```

Check the "Break the rules!" section hehe.

https://www.punctuationmatters.com/en-dash-em-dash-hyphen/